components/podio/actions/create-task/create-task.mjs

```diff
@@ -0,0 +1,60 @@
+import app from "../../podio.app.mjs";
+
+export default {
+  type: "action",
+  key: "podio-create-task",
+  version: "0.0.1",
+  name: "Create Task",
+  description: "Creates a status to the given workspace. [See the docs](https://developers.podio.com/doc/tasks/create-task-22419)",
+  props: {
+    app,
+    text: {
+      type: "string",
+      label: "Text",
+      description: "The text of the task",
+    },
+    description: {
+      type: "string",
+      label: "Description",
+      description: "The description of the task",
+      optional: true,
+    },
+    responsible: {
+      type: "integer[]",
+      label: "Responsible",
+      description: "The contact id(s) responsible for the task",
+      optional: true,
+    },
```

Comment:
Would it be possible to use async options for the `responsible` prop? https://developers.podio.com/doc/contacts/get-contacts-22400

Reply:
Unfortunately there is no proper way to do that :/